components/elm/bld/namelist_files/namelist_defaults.xml

```diff
@@ -592,6 +592,7 @@ this mask will have smb calculated over the entire global land surface
 <!-- MALI grids that could be used if the glc_grid logic was working. -->
```

Comment:
@stephenprice  or @jonbob, do you know what this comment is referring to?

Reply:
I don't know about this one. Blame has Jon making this comment in 2018.